src/cisco_gnmi/nx.py

```diff
@@ -154,19 +288,18 @@ def subscribe_xpaths(
 
     def parse_xpath_to_gnmi_path(self, xpath, origin=None):
         """Attempts to determine whether origin should be YANG (device) or DME.
-        Errors on OpenConfig until support is present.
         """
-        if xpath.startswith("openconfig"):
-            raise NotImplementedError(
-                "OpenConfig data models not yet supported on NX-OS!"
-            )
         if origin is None:
             if any(
-                map(xpath.startswith, ["Cisco-NX-OS-device", "/Cisco-NX-OS-device"])
+                map(xpath.startswith, [
```

Comment:
Does the module name vary like this?

Reply:
For NX, the module name is capitalized but the namespace is not.  For XE and XR, the module name and namespace match case.  We build messages with only the namespace information available.

Reply:
Does NX respond to any/all of these cases? I simply haven't verified or have one running to try.